pr-checklist.md

```diff
@@ -0,0 +1,44 @@
+# Documentation checklist
+
+Before requesting a PR review, please check your work against this list of commonly missed items from the style guide. 
+
+Guidance for Grammarly, Screenshots, and Diagrams can be found in the [Docs Standards](https://papermoon.atlassian.net/wiki/spaces/DR/pages/885261/Docs+Standards){target=\_blank}
```

Comment:
Here you are linking to a confluence page which is not public. Maybe we should migrate that to this Github?